pkg/apis/kops/cluster.go

```diff
@@ -526,6 +526,7 @@ func (c *Cluster) FillDefaults() error {
 	} else if c.Spec.Networking.Canal != nil {
 		// OK
 	} else if c.Spec.Networking.Kuberouter != nil {
+		c.Spec.Networking.Kuberouter.LivenessProbeEnabled = true
```

Comment:
For something that defaults to true, it's probably better to have the field be a `*bool` - or we could have the default be false (by inverting the meaning of the field).  The problem is that with upgrades from existing clusters, it is hard to know whether a false means "not set" or whether it actually means "no".